### PR TITLE
Operator Api: add show endpoint for product

### PR DESCRIPTION
**Pull request by tom-ioki**
This PR introduces the `show` endpoint for the Operator API.

### CODE REVIEW

File: lib/ioki/apis/operator_api.rb [hunk 1]
```diff
@@ -12,6 +12,11 @@ class OperatorApi
         base_path:   [API_BASE_PATH],
         model_class: Ioki::Model::Operator::Product
       ),
+      Endpoints::Show.new(
+        :product,
+        base_path:   [API_BASE_PATH],
+        model_class: Ioki::Model::Operator::Product
+      ),
```
Comment (eikes) [inline lib/ioki/apis/operator_api.rb:19]: Looks good, you could also do this:
```ruby
      Endpoints.crud_endpoints(
        :product,
        base_path:   [API_BASE_PATH],
        model_class: Ioki::Model::Operator::Product,
        except:      [:create, :update, :delete]
      ),
```

**Comment by tom-ioki** [comment]
@eikes could you have another look at this please 🙏  Thx

**Comment by eikes** [comment]
Looks good to me!

**Comment by eikes** [comment]
For extra credit: Add some tests!

**Comment by tom-ioki** [comment]
Sorry for the late response.

Not sure if it makes that much sense to have specs for this change. The `OperatorApi` class contains two constants and that's it. Also testing for the content of the `ENDPOINTS` constant is rather tedious and impractical.

Other than that, there are no specs for `DriverApi`, `PassengerApi` and `PlatformApi` 🤷 

Is it fine, if I skip the specs for this?

**Comment by eikes** [comment]
I was referring to tests like these:

https://github.com/ioki-mobility/ioki-ruby/blob/main/spec/ioki/platform_api_spec.rb#L51-L73

**Comment by eikes** [comment]
I see that the Operator API has no tests yet, so it's a good time as any to create that file. If you don't find the time, we should at least create an issue for that.

**Comment by tom-ioki** [comment]
Ah I see! That's where the specs are :D I was looking for them in `/spec/ioki/apis/`
Alright, I will add some specs then!

**Comment by eikes** [comment]
Looks superbe to me! Thanks for adding that spec file!

**Comment by tom-ioki** [comment]
Awesome! Thank you for the review! Can I merge it then?

**Comment by eikes** [comment]
Yes!

**Pull request merged**